lib/queries/useInstanceDateTypes/useInstanceDateTypes.js

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+} from '@folio/stripes/core';
+import {
+  CQL_FIND_ALL,
+  LIMIT_MAX,
+} from '../../constants';
+
+export const useInstanceDateTypes = () => {
+  const ky = useOkapiKy();
+  const [namespace] = useNamespace({ key: 'instance-date-types' });
+
+  const query = useQuery({
+    queryKey: [namespace, 'instance-date-types'],
+    queryFn: () => ky.get(`instance-date-types?limit=${LIMIT_MAX}&query=${CQL_FIND_ALL}`).json(),

```

Comment:
I think we also need to add the interface for this API.

Reply:
Thanks for noticing. I also moved `settings` interface from `optionalOkapiInterfaces` to `okapiInterfaces` because the request to mod-settings will always be made, which makes it a required interface